### PR TITLE
[WFLY-16267] Upgrade bouncycastle to 1.71

### PR DESCRIPTION
**Pull request by boris-unckel**
Fixes https://issues.redhat.com/browse/WFLY-16267

### CODE REVIEW

**Comment by boris-unckel** [comment]
The component upgrade of bouncycastle to 1.71 includes a maven coordinate change. Please find all the links in the Jira. This PR requires https://github.com/wildfly/wildfly-core/pull/5051 The discussion how to deal with it starts there.

**Review by yersan — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 1]
```diff
@@ -9941,6 +9941,37 @@
...
+                <version>1.69</version>
+                <scope>test</scope>
+            </dependency>
+
+            <dependency>
+                <groupId>org.bouncycastle</groupId>
+                <artifactId>bcutil-jdk15on</artifactId>
+                <version>1.69</version>
+                <scope>test</scope>
+            </dependency>
```
[the remark this answers is not in the document]
Comment (boris-unckel) [reply pom.xml]: @yersan Thanks for the feedback and explanation. I have changed it accordingly.

**Review by yersan — APPROVED** [review]
Looks good to me.

**Comment by jmesnil** [comment]
@boris-unckel   This PR is superseded by #15484 where I cherry-picked your commit. Thanks!

**Pull request closed without merging**